src/hidpp20.h

```diff
@@ -317,6 +317,34 @@ union hidpp20_button_binding {
 } __attribute__((packed));
 _Static_assert(sizeof(union hidpp20_button_binding) == 4, "Invalid size");
 
+struct hidpp20_color {
+	uint8_t red;
+	uint8_t green;
+	uint8_t blue;
+};
```

Comment:
I think this one needs to be packed too? might not matter if we're only using it inside another packed struct, I'm not sure